common/Android.mk

```diff
@@ -62,6 +62,11 @@ LOCAL_CPPFLAGS += \
 	-DDISABLE_CURSOR_PLANE
 endif
 
+ifeq ($(strip $(ENABLE_VA_DRIVER)), false)
+LOCAL_CPPFLAGS += \
```

Comment:
Yes, suggest to remove rarenderer.cpp and other VPP source code from being compiled if DISABLE_VA is set.
and please note that DISABLE_VA should be set to false in default!

Reply:
By default DISABLE_VA will not be defined untill ENABLE_VA_DRIVER is configured to false on certain specific platform. 

There is still much code coupled with VA in other segments rather than va files. Removing all va relevant stuff may need sort of code restructure. How about let this patch in first to resolve display issue when playback for platform that video driver is still not integrated, and create new issue to remove all va related stuff under this option?

Reply:
ENABLE_VA_DRIVER where we fine this? Is it in the mixins? I saw only INTEL_VA=true

Reply:
It needs to define in BoardConfig which I included in another 1a patch.

Reply:
Could you also open a PR for the BoardConfig change here: https://github.com/projectceladon/device-androidia-mixins